.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  hugo: circleci/hugo@0.5.2
-  node: circleci/node@5.1.0
+  hugo: circleci/hugo@1.3.1
+  node: circleci/node@5.2.0
```

Comment:
@clmedders @nick-mon1 we should check in on these versions in POAM.

Reply:
will add in poam doc to capture this!

Reply:
Thank you!